RecoEgamma/EgammaElectronProducers/plugins/GEDGsfElectronProducer.cc

```diff
@@ -134,3 +131,24 @@ void GEDGsfElectronProducer::matchWithPFCandidates(edm::Event & event)
     }
   }
 }
+
+void GEDGsfElectronProducer::setMVAOutputs(reco::GsfElectronCollection & electrons,
+                                    const gsfAlgoHelpers::HeavyObjectCache* hoc,
+                                    const std::map<reco::GsfTrackRef,reco::GsfElectron::MvaOutput> & mvaOutputs,
+                                    reco::VertexCollection const& vertices) const
+{
+  for( auto el = electrons.begin() ; el != electrons.end() ; el++ ) {
+    if(strategyCfg_.gedElectronMode==true) {
```

Comment:
Is this check still needed, as you have now moved the method inside the ```GEDGsfElectronProducer``` class?

Reply:
Thanks for catching that! I guess it's not needed and being able to drop these kind of checks was exactly one of the purposes of this PR.

Reply:
Besides the clang-format, I also removed that check.